sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/AzureCliScript.cs

```diff
@@ -77,9 +82,11 @@ public AzureCliScript()
         /// changed. Can be current time stamp or a GUID.</param>
         /// <param name="timeout">Maximum allowed script execution time
         /// specified in ISO 8601 format. Default value is PT1H</param>
-        public AzureCliScript(ManagedServiceIdentity identity, string location, System.TimeSpan retentionInterval, string azCliVersion, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string cleanupPreference = default(string), string provisioningState = default(string), ScriptStatus status = default(ScriptStatus), IDictionary<string, object> outputs = default(IDictionary<string, object>), string primaryScriptUri = default(string), IList<string> supportingScriptUris = default(IList<string>), string scriptContent = default(string), string arguments = default(string), IList<EnvironmentVariable> environmentVariables = default(IList<EnvironmentVariable>), string forceUpdateTag = default(string), System.TimeSpan? timeout = default(System.TimeSpan?))
-            : base(identity, location, id, name, type, tags)
+        public AzureCliScript(string location, System.TimeSpan retentionInterval, string azCliVersion, string id = default(string), string name = default(string), string type = default(string), ManagedServiceIdentity identity = default(ManagedServiceIdentity), IDictionary<string, string> tags = default(IDictionary<string, string>), SystemData systemData = default(SystemData), ContainerConfiguration containerSettings = default(ContainerConfiguration), StorageAccountConfiguration storageAccountSettings = default(StorageAccountConfiguration), string cleanupPreference = default(string), string provisioningState = default(string), ScriptStatus status = default(ScriptStatus), IDictionary<string, object> outputs = default(IDictionary<string, object>), string primaryScriptUri = default(string), IList<string> supportingScriptUris = default(IList<string>), string scriptContent = default(string), string arguments = default(string), IList<EnvironmentVariable> environmentVariables = default(IList<EnvironmentVariable>), string forceUpdateTag = default(string), System.TimeSpan? timeout = default(System.TimeSpan?))
```

Comment:
This is a breaking change. Why was the order changed? Can you add related swagger PR link to the PR description?

Reply:
Sure, here's the PR:

https://github.com/Azure/azure-rest-api-specs/pull/10784

This new api-version makes ManagedIdentity an optional argument, if the ManagedIdentity is not passed we use the user's token to create the supporting resources by leveraging ARM OBO token.